### ISSUE TITLE
Contents of blog/ in the homepage (content/_index.md)

### ISSUE THREAD

**Issue by samueloph** [Jieiku/abridge#168]
Looks like the main index at `content/_index.md` gets populated with the posts that are kept under `archive/`.

I've disabled the archive and kept only `blog/`, since I want the posts to be namespaced: https://github.com/samueloph/samueloph.github.io/blob/eccfd1c4b96dd67c6d662acebb8bb2ab3c156d43/config.toml#L81-L83

But then the homepage ends up empty. As a workaround I setup a redirect to `blog/` but that means every access gets a redirect by default.

Is there a way to have the contents of blog to show up in the homepage?

**Comment by Jieiku** [comment]
The Abridge demo has a blog "section"

https://abridge.netlify.app/blog/

This was tackled here: https://github.com/Jieiku/abridge/issues/128#issuecomment-1640844439 (I just noticed you opened that one too)

You will see that the blog section is an empty page in the demo, but that is only because the theme does not have any content stored in the blog directory, but If I move content to that directory then it would appear:

![2024-04-17_12-55-25](https://github.com/Jieiku/abridge/assets/106644/88a66783-6e0a-44c5-8066-a6127afe42df)

![2024-04-17_12-56-08](https://github.com/Jieiku/abridge/assets/106644/2168fb5f-9d79-4f8f-b299-3503ed587179)

![2024-04-17_12-57-26](https://github.com/Jieiku/abridge/assets/106644/e1bce0ac-1190-4524-bbf5-41e240d38ffc)

Also Take a look at the solution proposed by **cskwrd** in this thread:
https://github.com/Jieiku/abridge/issues/97

**Comment by samueloph** [comment]
Thank you for the quick response!!!

The contents under `blog/` are showing up fine when accessing that route, but on the main page I still get nothing.
Looks like there's something blocking their contents of appearing at the root index.

In the example you showed, do you get anything on the homepage?

Here's what I have:
![Screenshot from 2024-04-17 21-13-33](https://github.com/Jieiku/abridge/assets/2778482/ef3240af-ecc1-4b70-a11f-87b7ef46af2b)

While the page `blog/` looks fine:
![Screenshot from 2024-04-17 21-13-38](https://github.com/Jieiku/abridge/assets/2778482/b46b5501-f653-4c62-a12e-7d56afbfefc1)

The screenshots above are from the same website, with these settings:
![Screenshot from 2024-04-17 21-10-56](https://github.com/Jieiku/abridge/assets/2778482/a9dece34-e46b-4466-8eed-a10ee1303475)
![Screenshot from 2024-04-17 21-11-33](https://github.com/Jieiku/abridge/assets/2778482/2db3b711-e909-4910-82b0-ed6e57e44448)
![Screenshot from 2024-04-17 21-13-03](https://github.com/Jieiku/abridge/assets/2778482/3856d5a6-6e30-4853-b295-53382f139331)

So in other words, I'm looking for how to get the `blog/` posts to also show up at the root index.

**Comment by Jieiku** [comment]
If I move the content from the main content directory to under /blog then yes the main index is blank.

You can solve this by turning /blog into a transparent section, which makes all of the content of /blog available under the main site index.

You would do this here: your_site/content/blog/_index.md

instead of this:

```
+++
paginate_by = 3
sort_by = "date"
template = "index.html"
+++
```

set to this:

```
+++
transparent = true
+++
```

I would read over this thread as it is very similar: https://github.com/Jieiku/abridge/issues/97

**Comment by Jieiku** [comment]
I was just thinking over this in my head. In the case of wanting content to appear at both /blog and the main index.

I think one way to do what your wanting, if transparent section is not what your looking for. Is to modify the Template for the main site index to also include specified sections.

I think this is how tabi displays content from "blog" on their main home page: https://welpo.github.io/tabi/

If that is the solution your looking for (take a look at their demo and see if it is) then let me know.

**Comment by Jieiku** [comment]
I just looked back on an issue here https://github.com/Jieiku/abridge/issues/129 so I do believe it should be possible, so let me know if a transparent section is not what your looking for and I will review this again, my memory on it is a bit fuzzy.

**Comment by samueloph** [comment]
Looks like I just needed to set `blog/_index.md` to transparent, I initially thought that would remove the `blog/` prefix from the URLs but that's not the case.

https://github.com/samueloph/samueloph.github.io/commit/eac9dd1cb77d839327fea218f05bdf05f8261cc8

Thank you!
